### PR TITLE
Feature/merge terrains

### PR DESCRIPTION
**Pull request by Zakhar491**

### CODE REVIEW

File: Assets/New Terrain 1.asset
Comment (TimHubDevs) [inline Assets/New Terrain 1.asset:1]: Change name file

File: Assets/New Terrain.asset
Comment (TimHubDevs) [inline Assets/New Terrain.asset:1]: Change name file

File: Assets/New.meta
Comment (TimHubDevs) [inline Assets/New.meta:1]: Change name folder

File: Assets/New/New Terrain.asset
Comment (TimHubDevs) [inline Assets/New/New Terrain.asset:1]: Change name file

File: Assets/New/TerrainData_0d7bbf2a-5ebc-414a-841b-516c9013b726.asset
Comment (TimHubDevs) [inline Assets/New/TerrainData_0d7bbf2a-5ebc-414a-841b-516c9013b726.asset:1]: Create folder terrain data and move it to here

File: Assets/Rock Package.meta
Comment (TimHubDevs) [inline Assets/Rock Package.meta:1]: Move it to 3d party

File: Assets/Terrain/ForestTerrain.asset
Comment (TimHubDevs) [inline Assets/Terrain/ForestTerrain.asset:1]: delete old terrain and old data

File: Assets/Third party/Rock Package.meta
Comment (TimHubDevs) [inline Assets/Third party/Rock Package.meta:1]: Duplicate? Delete dublicat

File: ProjectSettings/TimelineSettings.asset
Comment (TimHubDevs) [inline ProjectSettings/TimelineSettings.asset:1]: delete

**Review by TimHubDevs — CHANGES_REQUESTED** [review]

File: Assets/New Terrain 1.asset
Comment (Zakhar491) [reply Assets/New Terrain 1.asset:1]: Done

File: Assets/New Terrain.asset
Comment (Zakhar491) [reply Assets/New Terrain.asset:1]: Done

**Review by TimHubDevs — APPROVED** [review]

File: Assets/New.meta
Comment (Zakhar491) [reply Assets/New.meta:1]: Done

File: Assets/New/New Terrain.asset
Comment (Zakhar491) [reply Assets/New/New Terrain.asset:1]: Done

File: Assets/New/TerrainData_0d7bbf2a-5ebc-414a-841b-516c9013b726.asset
Comment (Zakhar491) [reply Assets/New/TerrainData_0d7bbf2a-5ebc-414a-841b-516c9013b726.asset:1]: Done

File: Assets/Rock Package.meta
Comment (Zakhar491) [reply Assets/Rock Package.meta:1]: Done

**Pull request merged**